snuba/web/query.py

```diff
@@ -172,12 +172,14 @@ def _format_storage_query_and_run(
     ) as span:
         span.set_tag("table", source)
         try:
-            span.set_tag(
-                "ast_query",
-                AstSqlQuery(clickhouse_query, request_settings).format_sql(),
+            span.set_data(
+                "ast_query", AstSqlQuery(clickhouse_query, request_settings).sql_data()
             )
+            span.set_tag("query_type", "ast")
         except Exception:
-            logger.warning("Failed to format ast query", exc_info=True)
```

Comment:
This is important, we should keep it. Logging a warning shows up in our sentry issue dashboard with a lot of troubleshooting data.  That's what we use today to see if the new parser is failing.

Reply:
Unintentional removal. Re-adding